js/index.js

```diff
@@ -40,3 +40,60 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navContent = document.getElementsByTagName("a");
+navContent[0].textContent = siteContent.nav["nav-item-1"];
+navContent[1].textContent = siteContent.nav["nav-item-2"];
+navContent[2].textContent = siteContent.nav["nav-item-3"];
+navContent[3].textContent = siteContent.nav["nav-item-4"];
+navContent[4].textContent = siteContent.nav["nav-item-5"];
+navContent[5].textContent = siteContent.nav["nav-item-6"];
```

Comment:
Remember, `getElementsBy<whatever>` return an HTML Collection. So as an alternative, you can iterate over HTML Collections by converting them into full arrays like this:

```js
navContent = Array.from(navContent);
navContent.forEach(
  (element, i) => element.textContent = siteContent.nav[`nav-item-${i + 1}`]
);
```

If you do `querySelectorAll` instead, you'll get back a `NodeList`. If you want to do `forEach` on that, it is not necessary to convert it into an array first (e.g. `Array.from(whatever)`); it already has the `forEach` method baked in.

While the application here might a little interesting, you can definitely do this for easier things, like, you know, _changing the color of all the navigation links green_. :grin: